ps_checkout.php

```diff
@@ -137,11 +137,14 @@ public function __construct()
      */
     public function install()
     {
+        $webhookHelper = new PrestaShop\AccountsAuth\Wrapper\Webhook();
```

Comment:
La lib est-elle bien mise en dep, et publiée ? Prérequis !

Reply:
yep https://github.com/PrestaShopCorp/prestashop_accounts_auth/tree/feature/790-extract-library-from-ps_accounts

Reply:
To make sure we may test it, the lib should be added into the composer.json